src/lib/components/composites/paper-components/PaperListEntry.svelte

```diff
@@ -77,14 +97,16 @@ Usage:
 >
     <div
         class="flex flex-auto {showReviewStatus
-            ? `border-l-4 ${reviewDecisionColor[paper.reviewData?.finalDecision ?? 'unreviewed']}`
+            ? `border-l-4 ${getReviewDecisionColor(projectPaper.decision, projectPaper.reviews.length)}`
             : ''} rounded-md px-3 py-1.5"
     >
-        <PaperInfo {paper} />
+        <PaperInfo paper={projectPaper.paper!} />
     </div>
-    {#if paper.reviewData !== undefined && showReviewStatus}
-        {#each paper.reviewData.reviews as review}
-            <UserAvatar user={review.user} reviewDecision={review.decision} />
+    {#if projectPaper.reviews.length !== 0 && showReviewStatus}
```

Comment:
`projectPaper.reviews.length !== 0` is not necessary because the `#each` block doesn't get executed when there are no reviews
```suggestion
    {#if showReviewStatus}
```

Reply:
You are right, I think I only updated the old code, where this check was still necessary. Thank you for this hint.